Dockerfile

```diff
@@ -44,21 +42,10 @@ RUN yum --enablerepo=extras install epel-release -y && \
       pip install awscli
 
 RUN useradd nginx && \
-    mkdir /usr/local/openresty/naxsi/locations && \
-    mkdir /usr/local/openresty/nginx/client_body_temp && \
-    mkdir /usr/local/openresty/nginx/proxy_temp && \
-    mkdir /usr/local/openresty/nginx/fastcgi_temp && \
-    mkdir /usr/local/openresty/nginx/uwsgi_temp && \
-    mkdir /usr/local/openresty/nginx/scgi_temp && \
-    chown -R nginx:nginx /usr/local/openresty/naxsi/locations && \
-    chown -R nginx:nginx /usr/local/openresty/nginx/conf && \
-    chown -R nginx:nginx /usr/local/openresty/nginx/logs && \
-    chown -R nginx:nginx /usr/local/openresty/nginx/client_body_temp && \
-    chown -R nginx:nginx /usr/local/openresty/nginx/proxy_temp && \
-    chown -R nginx:nginx /usr/local/openresty/nginx/fastcgi_temp && \
-    chown -R nginx:nginx /usr/local/openresty/nginx/uwsgi_temp && \
-    chown -R nginx:nginx /usr/local/openresty/nginx/scgi_temp && \
-    chown -R nginx:nginx /usr/share/GeoIP
+    install -o nginx -g nginx -d /usr/local/openresty/naxsi/locations \
+                                 /usr/local/openresty/nginx/{client_body,fastcgi,proxy,scgi,uwsgi}_temp && \
```

Comment:
This lines doesn't seem too make directories like I expect it to do from previous behaviour.